ui/admin/app/controllers/scopes/scope/groups/group/index.js

```diff
@@ -0,0 +1,5 @@
+import Controller, { inject as controller } from '@ember/controller';
+
+export default class ScopesScopeGroupsGroupIndexController extends Controller {
+  @controller('scopes/scope/groups/index') groups;
```

Comment:
None blocking question: I do understand this line injects the groups contoller (`ui/admin/app/controllers/scopes/scope/groups/index.js` is that correct?